src/controllers/home.js

```diff
@@ -1,45 +1,41 @@
-
-const bcrypt = require('bcrypt');
-const   getUser = require('./../database/queries/get_login');
-const selectAllProject = require('./../database/queries/selectAllproject');
-const { sign, verify } = require('jsonwebtoken');
-const SECRET = 'poiugyfguhijokpkoihugyfyguhijo';
-
+const bcrypt = require("bcrypt");
+const getUser = require("./../database/queries/get_login");
+const selectAllProject = require("./../database/queries/selectAllproject");
+const { sign, verify } = require("jsonwebtoken");
+const SECRET = "poiugyfguhijokpkoihugyfyguhijo";
```

Comment:
This should never be here
It should be in the config.env which shouldn't be visible.
change it

Reply:
that fixed in the latest pull